multigeiger/multigeiger.ino

```diff
@@ -104,7 +106,16 @@ int update_wifi_status(void) {
   return st;
 }
 
-void display(unsigned long current_ms, unsigned long current_counts, unsigned long gm_count_timestamp, unsigned long current_hv_pulses) {
+int update_ble_status(void) {  // currently no error detection
+  int st = ST_BT_OFF;
+  if (get_status(STATUS_BT) != ST_BT_OFF) {
+    st = is_ble_connected() ? ST_BT_CONNECTED : ST_BT_CONNECTABLE;
+    set_status(STATUS_BT, st);
+  }
+  return st;
+}
```

Comment:
The reverse reading via get_status() is a bit strange.

Couldn't we do it just like:
```
int st;
if (sendToBle)
  st = is_ble_connected() ? ST_BT_CONNECTED : ST_BT_CONNECTABLE;
else
  st = ST_BT_OFF;
set_status(STATUS_BT, st);
return st;
```

Reply:
Right. I wanted to avoid a status line update every time we run the main loop, but the set_status() is indeed smart enough to update only when actually changed. Will revert here.
Another PR can then indeed remove the get_status() function from display.cpp, as it was only needed here...